src/services/__mocks__/usersService.ts

```diff
@@ -74,7 +74,7 @@ export const mockedUsers: Array<PartyUser> = [
           {
             relationshipId: 'rel2',
             role: 'referente-tecnico',
-            selcRole: 'LIMITED',
+            selcRole: 'ADMIN',

```

Comment:
This should be LIMITED in order to have an example of product LIMITED for the current user